sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/query.sql

```diff
@@ -0,0 +1,52 @@
+-- Aggregated users data joined with Country Lookup
+SELECT
+  submission_date,
+  country.name AS country,
+  city,
+  activity_segment AS segment,
+  normalized_app_name AS app_name,
+  app_version AS app_version,
+  normalized_channel AS channel,
+  distribution_id AS distribution,
+  normalized_os AS os,
+  normalized_os_version AS os_version,
+  os_version_major,
+  os_version_minor,
+  locale AS language,
+  attribution_content,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_experiment,
+  attribution_variation,
+  COUNT(DISTINCT client_id) AS mau,
+  APPROX_COUNT_DISTINCT(IF(days_since_seen < 7, client_id, NULL)) AS wau,
```

Comment:
Shouldn't we do a COUNT DISTINCT instead of an APPROX COUNT as this will be final derived tables? I think the variances between two runs will probably be small using approx count but won't we add some overhead to whoever is doing analysis to have to account for that should they need to go upstream from the derived to the client level data.

Reply:
Wow wow of course (it was a test on numbers)... thanks for the good spotting!